js/terms.json

```diff
@@ -136,52 +131,320 @@
+  },
+  {
+    "term": "Act",
+    "definition": "The Federal Election Campaign Act of 1971, as amended (52 U.S.C. §§30101-30146). 11 CFR 100.18. Prior to September 1, 2014, the Act appeared in Title 2 of the U.S. Code. Sometimes abbreviated FECA."
+  },
+  {
+    "term": "Administrative expense",
+    "definition": "For party committees, rent, utilities, office equipment, office supplies, routine building maintenance and other operating costs not attributable to a specific candidate."
+  },
+  {
+    "term": "Advisory opinion (AO)",
+    "definition": "A formal response from the Commission regarding the legality of a specific activity proposed in an advisory opinion request (AOR). 11 CFR Part 112."
+  },
+  {
+    "term": "Affiliated committees",
+    "definition": "Committees and organizations that are considered one committee for purposes of the contribution limits. 11 CFR 110.3(a)(1). Affiliated committees include (1) All committees established or authorized by a candidate as part of his or her campaign for federal or nonfederal office; and (2) All committees established, financed, maintained or controlled by the same person, group or organization. 11 CFR 100.5(g)(1) and (2); 110.3(a)(1)."
+  },
+  {
+    "term": "Allocation account",
+    "definition": "A separate federal account into which funds from either a committee’s federal and nonfederal accounts, or (for party committees) from its federal and Levin accounts, are deposited solely to pay expenses that must be allocated. (A party committee must have separate allocation accounts for its federal/nonfederal allocation and for its federal/Levin allocation.) 11 CFR 106.7(f) and 300.33(d)."
+  },
+  {
+    "term": "Bundled contribution",
+    "definition": "A contribution forwarded to a reporting committee by a lobbyist/registrant or lobbyist/registrant PAC, or received by a reporting committee and credited to a lobbyist/registrant or lobbyist/registrant PAC. See 11 CFR 104.22(a)(6)."
+  },
+  {
+    "term": "Campaign traveler",
+    "definition": "Any candidate traveling in connection with an election for federal office, or any individual traveling in connection with an election for federal office on behalf of a candidate or political committee; or any member of the news media traveling with a candidate. 11 CFR 100.93(a)(3)(i)."
+  },
+  {
+    "term": "Clearly identified candidate",
+    "definition": "A candidate is clearly identified when his or her name, nickname, photograph or drawing appears, or when his or her identity is otherwise apparent through an unambiguous reference such as “the President,” “your Congressman,” or “the incumbent,” or through an unambiguous reference to his or her status as a candidate such as “the Democratic presidential nominee” or “the Republican candidate for Senate in the State of Georgia.” 11 CFR 100.17."
```

Comment:
Eagle eye! So, typically yes. But it seems this is ok because these are smart quotes, and so they're not conflicting with the quotes that wrap the item.